examples/unsubscribe.ts

```diff
@@ -1,16 +1,16 @@
 import dayjs from "dayjs";
 import utc from "dayjs/plugin/utc";
 
-import { rest } from "yajsapi";
+import { rest } from "../yajsapi";
```

Comment:
again, doesn't it mean that we're depending on the source version of yajsapi?

Reply:
yes, you are right, i'll fix it.